crates/rb-sys-build/src/cc.rs

```diff
@@ -270,7 +273,13 @@ fn get_tool(env_var: &str, default: &str) -> Command {
     let mut tool_args = shellsplit(tool_args).into_iter();
     let tool = tool_args.next().unwrap_or_else(|| default.to_string());
 
-    let mut cmd = new_command(&tool);
+    let mut cmd = if !Path::new(&tool).is_file() {
```

Comment:
This is the fix

Reply:
imho I prefer to avoid negation in conditionals if possible, so instead I would go with 
```rust
let mut cmd = if Path::new(&tool).is_file() { 
    new_command(&tool)
} else {
    debug_log!("[WARN] {tool} tool not found, falling back to {default}");
    new_command(default)
}
```

feels easier to read/undertsand, but just an opinion xD

Reply:
done!